lib/shopify_api/resources/ping/message.rb

```diff
@@ -1,9 +1,23 @@
 # frozen_string_literal: true
 
+require 'byebug'
+
 module ShopifyAPI
   module Ping
     class Message < Base
       self.prefix = "/admin/api/ping-api/v1/conversations/:conversation_id/"
+
+      def successful_delivery(options)
```

Comment:
how does this work if you can't query for a message?

Reply:
While you can't query for a message, explicitly passing in `conversation_id` and `message_id` sets them as prefix values for the `delivery_confirmation` endpoint.